elements/rh-footer/rh-footer-lightdom.css

```diff
@@ -31,7 +31,7 @@
   display: block;
 }
 
-:is(rh-footer) a[slot^="logo"]>img {
+:is(rh-footer) a[slot^="logo"] > img {
```

Comment:
I submitted a fix for this linting error upstream: https://github.com/RedHat-UX/red-hat-design-system/pull/1777